model/commitq/githubpr/github_pr_message.go

```diff
@@ -0,0 +1,90 @@
+package githubpr
+
+import (
+	"fmt"
+
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+)
+
+type GithubMergeMethod string
+
+// valid Github merge methods
+const (
+	GithubMergeMethodMerge  = GithubMergeMethod("merge")
+	GithubMergeMethodSquash = GithubMergeMethod("squash")
+	GithubMergeMethodRebase = GithubMergeMethod("rebase")
+)
+
+type GithubMergePR struct {
+	ProjectID   string            `bson:"projectID,omitempty" json:"projectID,omitempty" yaml:"projectID,omitempty"`
+	Owner       string            `bson:"owner,omitempty" json:"owner,omitempty" yaml:"owner,omitempty"`
+	Repo        string            `bson:"repo,omitempty" json:"repo,omitempty" yaml:"repo,omitempty"`
+	Ref         string            `bson:"ref,omitempty" json:"ref,omitempty" yaml:"ref,omitempty"`
+	PRNum       int               `bson:"PR_num,omitempty" json:"PR_num,omitempty" yaml:"PR_num,omitempty"`
+	CommitMsg   string            `bson:"commit_msg,omitempty" json:"commit_msg,omitempty" yaml:"commit_msg,omitempty"`
+	CommitTitle string            `bson:"commit_title,omitempty" json:"commit_title,omitempty" yaml:"commit_title,omitempty"`
+	MergeMethod GithubMergeMethod `bson:"merge_method,omitempty" json:"merge_method,omitempty" yaml:"merge_method,omitempty"`
```

Comment:
why are these all omit empty?

I'd opt for longer names for CommitMessage (the field name,) and might chose different non-cap names for some of the other fields. also projectID rather than project_id is weird.

Reply:
what do you suggest making non-cap?

Reply:
Should be addressed now.